jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/CoreContextHandler.java

```diff
@@ -63,6 +62,7 @@ public class CoreContextHandler extends ContextHandler implements Deployable
 {
     private static final Logger LOG = LoggerFactory.getLogger(CoreContextHandler.class);
     private static final String ORIGINAL_BASE_RESOURCE = "org.eclipse.jetty.webapp.originalBaseResource";
+    private static final String[] SUPPORTED_ARCHIVE_EXTENSIONS = new String[]{"war", "jar", "zip"};
```

Comment:
This is in `core`, why does it refer to `.war` archives?

Reply:
It's just an archive format supported by core.
It was part of the list suggested by @sbordet